vtex-apps/yotpo
# Wrike 499536593: Improve timing of Yotpo widget refreshes

**What problem is this solving?**
When multiple Yotpo widgets are on the same page, strange behavior can be seen (main review widget contents end up inside a star ratings div, for example). To fix, this PR adds `setTimeout()` wrappers to any `yotpo.refreshWidgets()` calls.

**How should this be manually tested?**
New version linked here: https://yotpo--worldwidegolf.myvtex.com/proactive-sports-lite-stick-refill-for-glow-flyer-ball-10114014/p
Make sure to refresh page.

**Screenshots or example usage:**
Problem we are trying to avoid: 
![image (3)](https://user-images.githubusercontent.com/6306265/80511062-9560fa00-8949-11ea-99c8-14cdd4ca583f.png)
